tools/archives/pyvo_integration/astronomical_archives.xml

```diff
@@ -0,0 +1,1005 @@
+              <param name="service_type" type="select" label="Service type">
+                <option value="TAP" selected="true">TAP: Tables</option>
+                <option value="SCS">SCS: Cone search</option>
+                <option value="SSA">SSA: Spectra</option>
+                <option value="SIA">SIA: Images</option>
+              </param>
+              <param name="wavebands" type="select" label="Wavebands">
+                <option value="all" selected="true">All</option>
+                <option value="radio">Radio</option>
+                <option value="millimeter">Millimeter</option>
+                <option value="infrared">Infrared</option>
+                <option value="optical">Optical</option>
+                <option value="uv">UV</option>
+                <option value="euv">EUV</option>
+                <option value="x-ray">X-ray</option>
+                <option value="gamma-ray">Gamma-ray</option>
+              </param>
+            </when>
+            <when value="archive">
+              <param name="archive" type="select" label="Archive">
+                <options from_file="astronomical_archives_gen.loc">
+                  <column name="value" index="2" />
+                  <column name="name" index="1" />
+                </options>
+              </param>
+            </when>
+        </conditional>
+        <section name="query_section" title="Query selection" expanded="false">
+          <conditional name="query_selection">
+            <param name="query_type" type="select" label="ADQL Query Selection">
+              <option value="none">No specific query (first n files from archive)</option>
+              <option value="obscore_query">IVOA obscore table query builder</option>
+              <option value="raw_query">Raw ADQL query builder</option>
+            </param>
+            <when value="obscore_query">
+              <param name="target_name" type="text" label="Target name" help="Name of the astronomical object observed" />
+              <param name="t_min" type="text" label="Start time in MJD" />
+              <param name="t_max" type="text" label="Stop time in MJD" />
+              <param name="dataproduct_type" type="text" label="Data product type" help="Logical data product type (image etc.)" />
+              <param name="dataproduct_type_select" type="select" label="Data product type" help="Logical data product type">
+                <option value="none" selected="true">None</option>
+                <option value="image">Image</option>
+                <option value="cube">Cube</option>
+                <option value="dynamic-spectrum">Dynamic spectrum</option>
+                <option value="event-list">Event list</option>
+                <option value="light-curve">Light curve</option>
+                <option value="measurements">Measurements</option>
+                <option value="polarization-cube">Polarization cube</option>
```

Comment:
these two have the same value. In fact, it looks like only first one is shown in UI. when you put the reference for the list, we can check what to do best.

Reply:
I'm looking for the reference right now will put it as soon as I find it

Reply:
> these two have the same value. In fact, it looks like only first one is shown in UI. when you put the reference for the list, we can check what to do best.

finally found the list I used https://www.ivoa.net/rdf/product-type/2023-06-26/product-type.html#image

the double polarization-cube was an error from my part should have been "polarization-resolved-dataset". I have changed it in the code already

Reply:
> > these two have the same value. In fact, it looks like only first one is shown in UI. when you put the reference for the list, we can check what to do best.
> 
> finally found the list I used https://www.ivoa.net/rdf/product-type/2023-06-26/product-type.html#image

That's a good one, definitely put a reference to it. Note that there is also machine-readable rdf there which in principle can be used to generate the list you have here.

Reply:
But also put a reference to the pdf.